### ISSUE TITLE
DatePicker close modal after select day

### ISSUE THREAD

**Issue by ialex90** [ng-bootstrap/ng-bootstrap#1100]
Hi there,

I am using a date picker within a modal, both from this framework, and when I pick the day, the modal gets closed. Is there any reason in particular for this behavior? What I think is this click is closing the modal by backdrop.

I saw the backdrop could be configured but what I don't want is force to make the modal not closeable for backdrop events, is there any way to do it temporally? (for the after click event on the date picker)

Or any work around?

**EDIT**: I am using the date picker with popup (input with button and pop up)
**EDIT 2:** If I use an event and stopPropagation after the click on the day, it does not change the ngModel so it does not change the input day. But at least, it does not close the modal... maybe is related to the problem?

```
<template #customDay let-date="date" let-currentMonth="currentMonth" let-selected="selected" let-disabled="disabled">
     <span (click)="myEvent($event)" class="custom-day" [class.weekend]="isWeekend(date)" [class.bg-primary]="selected" [class.hidden]="date.month !== currentMonth" [class.text-muted]="disabled">
               {{ date.day }}
       </span>
</template>
```

Thanks!

**Comment by pkozlowski-opensource** [comment]
Please provide a minimal plunker with a reproduce scenario as requested in the issue template (https://github.com/ng-bootstrap/ng-bootstrap#you-think-youve-found-a-bug). A plunker is mandatory since we had a similar-sounding bug (#1042) which is fixed in 1.0.0-alpha.14.

tl;dr; try with 1.0.0-alpha.14 and if you still see the bug with the latest release please provide a plunker. Without a plunker it is not actionable and will be closed.

**Comment by ialex90** [comment]
I did not research on the issues before, sorry :(

It's fixed, I just upgraded the version and now it works perfectly, impressed with the time took you to reply @pkozlowski-opensource 

Thanks!
